package/MDAnalysis/lib/distances.py

```diff
@@ -129,7 +137,7 @@ def _run(funcname, args=None, kwargs=None, backend="serial"):
 from .c_distances_openmp import OPENMP_ENABLED as USED_OPENMP
 
 
-def _check_result_array(result, shape):
+def _check_result_array(result: np.ndarray, shape: tuple) -> np.ndarray:
```

Comment:
Because the minimum version of numpy is about to bump to 1.20.0, we went for `numpy.typing` to type numpy:

```python
import numpy.typing as not
from typing import Tuple

def _check_result_array(result: npt.NDArray, shape: Tuple) -> npt.NDArray:
```